WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -400,11 +400,6 @@ extension WordPressAppDelegate {
     }
 
     func configureSelfHostedChallengeHandler() {
-        /// Note:
-        /// WordPressKit, now imported via CocoaPods, has the `AppExtension Safe API Only` flag set to *true*. Meaning that
-        /// the host app is, effectively as of now, responsible for presenting any alert onscreen (whenever a HTTP Challenge is
-        /// received). Capicci?
-        ///
```

Comment:
This was a very old comment. It didn't seem to explain much to me, so I got rid of it.